schema.sql

```diff
@@ -1,12 +1,5 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals( 
-    id integer NOT NULL,
-    name varchar(100)
-    date_of_birth date,
-    escape_attempts integer,
-    neutered boolean,
-    weight_kg numeric,
-    CONSTRAINT animals_pkey PRIMARY KEY (id)
-)
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you don't remove the previous queries from your project because these queries are related to your next milestone queries so please don't delete them keep all the queries in your project 👍